libvast_test/CMakeLists.txt

```diff
@@ -26,23 +26,24 @@ target_sources(
   libvast_test
   INTERFACE
     "$<${isExe}:$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src/main.cpp>>"
-    "$<${isExe}:$<INSTALL_INTERFACE:$<INSTALL_PREFIX>/${CMAKE_INSTALL_DATADIR}/vast/test/main.cpp>>"
+    "$<${isExe}:$<INSTALL_INTERFACE:${_IMPORT_PREFIX}/${CMAKE_INSTALL_DATADIR}/vast/test/main.cpp>>"
```

Comment:
The old code here was correct, see https://cmake.org/cmake/help/v3.19/manual/cmake-generator-expressions.7.html#target-dependent-queries.

Please revert.